cuegui/cuegui/plugins/MonitorJobsPlugin.py

```diff
@@ -106,14 +108,36 @@ def getJobIds(self):
         return list(map(opencue.id, self.jobMonitor.getJobProxies()))
 
     def restoreJobIds(self, jobIds):
-        """Monitors a list of jobs."""
-        for jobId in jobIds:
-            try:
-                self.jobMonitor.addJob(jobId)
-            except opencue.EntityNotFoundException:
-                logger.warning("Unable to load previously loaded job since "
-                               "it was moved to the historical "
-                               "database: %s", jobId)
+        """Restore monitored jobs from previous saved state
+        Only load jobs that have a timestamp less than or equal to the time a job lives on the farm
+        (jobs are moved to historical database)
+
+        :param jobIds: monitored jobs ids and their timestamp from previous working state (loaded from config.ini file)
+        :ptype: list of tuples (ex: [("Job.f156be87-987a-48b9-b9da-774cd58674a3", 1612482716.170947),...
+        """
+        today = datetime.datetime.now()
+        limit = JOB_LOAD_LIMIT if len(jobIds) > JOB_LOAD_LIMIT else len(jobIds)
+        msg = """
```

Comment:
nit: For multiline strings I think we should use implicit joins instead of the triple quotes -- this adds extra whitespace to the log message, right?

Maybe something like:

```
msg = ('Unable to load previously loaded job since it was moved '
       'to the historical database: {0}')
```